2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -8,8 +8,8 @@ set -o pipefail
 # because mongodb.conf lives in /var/lib/mongodb/ and we don't want a volume to
 # override it.
 export MONGODB_DATADIR=/var/lib/mongodb/data
-export MONGODB_CONFIG_PATH=/etc/mongod.conf
-export MONGODB_KEYFILE_PATH="${HOME}/keyfile"
+MONGODB_CONFIG_PATH=/etc/mongod.conf
+MONGODB_KEYFILE_PATH="${HOME}/keyfile"
```

Comment:
How about organizing it such that exported and non-exported variables are separated?
I think it would be less puzzling for readers.